lib/backend/dynamo/dynamodbbk.go

```diff
@@ -99,6 +101,10 @@ func (cfg *Config) CheckAndSetDefaults() error {
 		return trace.BadParameter("DynamoDB: table_name is not specified")
 	}
 
+	if cfg.TenantId == "" {
+		cfg.TenantId = hashKey
+	}
```

Comment:
TenantId maps to what value in idemeum
is it the tenant name or tenant id( uuid is generated for a tenant).

TenantId needs to be injected from the teleport.yaml, it can not be hashKey.